starknet-foundry/crates/forge/src/main.rs

```diff
@@ -71,7 +71,8 @@ fn main_execution() -> Result<()> {
 
     for package in &scarb_metadata.workspace.members {
         let forge_config = forge::scarb::config_from_scarb_for_package(&scarb_metadata, package)?;
-        let (base_path, dependencies, target_name) =
+
+        let (base_path, _corelib_path, dependencies, target_name) =
```

Comment:
`_corelib_path` is unused for now, see the comment in `starknet-foundry/crates/forge/src/scarb.rs`